projects/demo/src/modules/markup/form/index.html

```diff
@@ -94,5 +100,11 @@
             [component]="1 | tuiComponent"
             [content]="1 | tuiExample"
         />
+        <tui-doc-example
```

Comment:
My idea was to recreate 1st example from this page in the layout form component on its page

Reply:
Okay! We need to move this example 2 to layouts form right?